# This file is distributed under the same license as the Django package.
#
# Translators:
# Bedilbek Khamidov <bedilbek@gmail.com>, 2019
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-09-29 11:40+0500\n"
"PO-Revision-Date: 2019-12-29 16:31+0000\n"
"Last-Translator: Bedilbek Khamidov <bedilbek@gmail.com>\n"
"Language-Team: Uzbek (http://www.transifex.com/django/django/language/uz/)\n"
"Language: uz\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: venv/lib/python3.8/site-packages/django/contrib/auth/admin.py:49
msgid "Personal info"
msgstr "Shaxsiy ma'lumotlar"

#: venv/lib/python3.8/site-packages/django/contrib/auth/admin.py:51
msgid "Permissions"
msgstr "Ruhsatnomalar"

#: venv/lib/python3.8/site-packages/django/contrib/auth/admin.py:62
msgid "Important dates"
msgstr "Muhim sanalar"

#: venv/lib/python3.8/site-packages/django/contrib/auth/admin.py:156
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "%(key)r asosiy kalitli %(name)s obyekt mavjud emas"

#: venv/lib/python3.8/site-packages/django/contrib/auth/admin.py:168
msgid "Password changed successfully."
msgstr "Parol muvaffaqiyatli o'zgartirildi."

#: venv/lib/python3.8/site-packages/django/contrib/auth/admin.py:189
#, python-format
msgid "Change password: %s"
msgstr "Parolni o'zgartirish: %s"

#: venv/lib/python3.8/site-packages/django/contrib/auth/apps.py:16
msgid "Authentication and Authorization"
msgstr "Autentifikatsiya va Avtorizatsiya"

#: venv/lib/python3.8/site-packages/django/contrib/auth/base_user.py:50
msgid "password"
msgstr "parol"

#: venv/lib/python3.8/site-packages/django/contrib/auth/base_user.py:51
msgid "last login"
msgstr "oxirgi kirish"

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:41
msgid "No password set."
msgstr "Hech qanday parol qo'yilmagan."

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:49
msgid "Invalid password format or unknown hashing algorithm."
msgstr "Noto'g'ri parol formati yoki noma'lum heshlash algoritmi ishlatilgan."

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:91
#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:360
#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:439
msgid "The two password fields didn’t match."
msgstr "Ikkala parol maydoni bir-biriga mos kelmadi."

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:94
#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:149
#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:182
#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:443
msgid "Password"
msgstr "Parol"

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:100
msgid "Password confirmation"
msgstr "Parolni tasdiqlash"

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:103
#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:454
msgid "Enter the same password as before, for verification."
msgstr "Tekshirish uchun avvalgi parolni kiriting.."

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:151
msgid ""
"Raw passwords are not stored, so there is no way to see this user’s "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""
"Xom parollar saqlanmaydi, shuning uchun bu foydanaluvchining parolini "
"ko'rishni iloji yo'q, lekin siz <a href=\"{}\"> shu formadan </a> foydalanib "
"parolni o'zgartirishingiz mumkin. "

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:189
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""
"Iltimos, to'g'ri %(username)s va parolni kiriting. Ahamiyat bering, ikkala "
"maydonlar ham katta-kichik harfga sezgir bo'lishi mumkin."

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:192
msgid "This account is inactive."
msgstr "Bu akkaunt nofaol"

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:257
msgid "Email"
msgstr "Elektron pochta"

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:363
msgid "New password"
msgstr "Yangi parol"

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:369
msgid "New password confirmation"
msgstr "Yangi parolni tasdiqlash"

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:407
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr ""
"Sizning eski parolingiz noto'g'ri kiritilgan edi. Iltimos, qaytadan kiriting."

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:411
msgid "Old password"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/forms.py:451
msgid "Password (again)"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:327
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:420
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:510
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:605
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:656
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:698
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:747
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:793
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:851
msgid "algorithm"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:328
msgid "iterations"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:329
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:426
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:512
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:609
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:657
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:699
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:852
msgid "salt"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:330
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:427
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:610
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:658
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:700
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:748
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:794
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:853
msgid "hash"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:421
msgid "variety"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:422
msgid "version"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:423
msgid "memory cost"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:424
msgid "time cost"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:425
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:608
msgid "parallelism"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:511
#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:606
msgid "work factor"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:513
msgid "checksum"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/hashers.py:607
msgid "block size"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:62
#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:116
msgid "name"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:66
msgid "content type"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:68
msgid "codename"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:73
msgid "permission"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:74
#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:119
msgid "permissions"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:126
msgid "group"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:127
#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:258
msgid "groups"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:249
msgid "superuser status"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:252
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:261
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:269
msgid "user permissions"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:271
msgid "Specific permissions for this user."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:345
msgid "username"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:349
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:353
msgid "A user with that username already exists."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:356
msgid "first name"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:357
msgid "last name"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:358
msgid "email address"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:360
msgid "staff status"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:362
msgid "Designates whether the user can log into this admin site."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:365
msgid "active"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:368
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:372
msgid "date joined"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:381
msgid "user"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/models.py:382
msgid "users"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:111
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:123
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:206
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:213
msgid "Your password can’t be too similar to your other personal information."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:245
msgid "This password is too common."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:250
msgid "Your password can’t be a commonly used password."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:261
msgid "This password is entirely numeric."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:266
msgid "Your password can’t be entirely numeric."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/templates/registration/password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/views.py:178
msgid "Logged out"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/views.py:237
msgid "Password reset"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/views.py:264
msgid "Password reset sent"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/views.py:274
msgid "Enter new password"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/views.py:346
msgid "Password reset unsuccessful"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/views.py:355
msgid "Password reset complete"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/views.py:367
msgid "Password change"
msgstr ""

#: venv/lib/python3.8/site-packages/django/contrib/auth/views.py:390
msgid "Password change successful"
msgstr ""
